HelloWorld/HelloWorld.cs

```diff
@@ -6,18 +6,74 @@ class Program
     {
         static void Main(string[] args)
         {
-            int [] numbers = {23, 7, 4, 3,11,8,9, 49};
-
-            if(numbers.Length ==0 ) {
-                Console.WriteLine("Your list is empty");
-            } else {
-                int largest = numbers[0];
-                for(int i=1;i<numbers.Length;i++){
-                    
-                }
-
-            }
-            }
+        string name = "";
```

Comment:
looks like this file has some issues, can you fix it and let me know when you have it done.